doc/api/tls.md

```diff
@@ -583,6 +583,20 @@ if called on a server socket. The supported types are `'DH'` and `'ECDH'`. The
 
 For Example: `{ type: 'ECDH', name: 'prime256v1', size: 256 }`
 
+### tlsSocket.getFinished()
+
+* Returns: {Buffer|undefined} The latest `Finished` message that has been
+sent to the socket as part of a SSL/TLS handshake, or `undefined` if
```

Comment:
The current implementation actually returns `null` in case there is no handle. Should this also be reflected here? And it would be good to add a test for that as well.

Reply:
@BridgeAR Actually, it might be more consistent to return `undefined` in case there is no handle. What do you think?

Reply:
I agree.